src/request.ts

```diff
@@ -14,25 +14,69 @@ export interface RequestOptions {
   tokenProvider?: TokenProvider;
 }
 
+export interface RawRequestOptions {
+  method: string;
+  url: string;
+  jwt?: string;
```

Comment:
I'd remove the `jwt` parameter to make it even less opinionated.

Reply:
Good point - will do.